pkg/agent/agent.go

```diff
@@ -178,9 +178,22 @@ func ParseDriverType(raw string) (StorageForDriver, error) {
 func appOverviewFromRelease(r *release.Release) proxy.AppOverview {
 	return proxy.AppOverview{
 		ReleaseName: r.Name,
-		Version:     strconv.Itoa(r.Version),
+		Version:     r.Chart.Metadata.Version,
 		Icon:        r.Chart.Metadata.Icon,
 		Namespace:   r.Namespace,
 		Status:      r.Info.Status.String(),
+		Chart:       r.Chart.Name(),
+		ChartMetadata: chartv1.Metadata{
+			Name:        r.Chart.Metadata.Name,
+			Home:        r.Chart.Metadata.Home,
+			Sources:     r.Chart.Metadata.Sources,
+			Version:     r.Chart.Metadata.Version,
+			Description: r.Chart.Metadata.Description,
+			Keywords:    r.Chart.Metadata.Keywords,
+			Icon:        r.Chart.Metadata.Icon,
+			ApiVersion:  r.Chart.Metadata.APIVersion,
+			Tags:        r.Chart.Metadata.Tags,
+			AppVersion:  r.Chart.Metadata.AppVersion,
+		},
```

Comment:
I think we could and should re-use [`compatibleMetadata`](https://github.com/kubeapps/kubeapps/blob/8bb060e15ad315c1561d0133cbd30827d292dcd2/cmd/kubeops/internal/handler/dashboardcompat.go#L40). I managed to make it typecheck with little effort at least.

This would of course mean said function would have to be moved to somewhere in `pkg`. At closer thought, maybe it makes more sense to put the entire `dashboardcompat.go` in `pkg`. It really is more of a package than a command.

Reply:
That's a good point. At the end we have a package to convert releases from one format to the other so we can reuse it.